app/bundles/EmailBundle/Config/config.php

```diff
@@ -383,14 +383,29 @@
                 'serviceAlias' => 'swiftmailer.mailer.transport.%s',
                 'arguments'    => [
                     '%mautic.mailer_amazon_region%',
-                    'mautic.http.connector',
+                    '%mautic.mailer_host%',
+                    '%mautic.mailer_port%',
+                    'mautic.transport.amazon.callback',
+                ],
+                'methodCalls' => [
+                    'setUsername' => ['%mautic.mailer_user%'],
+                    'setPassword' => ['%mautic.mailer_password%'],
+                ],
+            ],
+            'mautic.transport.amazon_api' => [
+                'class'        => 'Mautic\EmailBundle\Swiftmailer\Transport\AmazonApiTransport',
```

Comment:
Should be \Mautic\EmailBundle\Swiftmailer\Transport\AmazonApiTransport::class

Reply:
Thanks so much for your review, regards this edit i'm a bit confused since original amazon transport uses
`'Mautic\EmailBundle\Swiftmailer\Transport\AmazonTransport'`
and all other transports use the same format as well so i'm wondering if ther is a specific reason for your suggestion.

Reply:
It's just minor. Full class reference is better to read by IDE software and refactor it.

Reply:
Thanks Kuzmany, I just pushed your edits, Please let me know if you suggest any other edits.